Tests/TuistGeneratorIntegrationTests/Utils/Mocks/MockEmbedScriptGenerator.swift

```diff
@@ -0,0 +1,23 @@
+import Basic
+import Foundation
+
+@testable import TuistCoreTesting
+@testable import TuistGenerator
+
+final class MockEmbedScriptGenerator: EmbedScriptGenerating {
```

Comment:
Looks like we'll need another copy of this within `TuistGeneratorTests` (I believe one of the tests there references it too)